File: src/etch/bindings/java/support/FieldMap.java
package etch.bindings.java.support;

import etch.bindings.java.msg.Field;

/**
 * FieldMap is a IdNameMap for Field.
 */
public final class FieldMap extends IdNameMap<Field>
{
	@Override
	public Field makeNew( String name )
	{
		return new Field( name );
	}
}

File: src/etch/bindings/csharp/support/FieldMap.cs
using System;
using System.Collections.Generic;
using System.Text;
using Etch.Msg;

namespace Etch.Support
{
    /// <summary>
    /// FieldMap is an IdNameMap for Field
    /// </summary>
    public class FieldMap : IdNameMap<Field>
    {
        public override Field MakeNew( string name )
        {
            return new Field( name );
        }
    }
}
